conf/custom_logging.py

```diff
@@ -0,0 +1,26 @@
+from .base import LOGGING, LOG_LEVEL, PROFILES_ENABLED
+
```

Comment:
Generally I'd like all related config to be together. Maybe bring the definition for LOGGING over to this file?

Reply:
Also I'm wondering if there is any of the configuration below that can be added to the base LOGGING config - I know that we only want to enable this `if PROFILES_ENABLED`, but I think maybe some of the stuff can be _defined_ just not enabled. This will reduce the amount of branched code.